cas/remote.go

```diff
@@ -17,27 +17,35 @@ package cas
 import (
 	"encoding/json"
 	"fmt"
+	"io"
+	"io/ioutil"
 	"net/http"
 	"os"
 	"time"
 
+	"github.com/coreos/rocket/pkg/keystore"
+
+	"github.com/appc/spec/aci"
 	"github.com/appc/spec/schema/types"
 	"github.com/coreos/rocket/Godeps/_workspace/src/github.com/mitchellh/ioprogress"
+	"github.com/coreos/rocket/Godeps/_workspace/src/golang.org/x/crypto/openpgp"
 )
 
-func NewRemote(name string, mirrors []string) *Remote {
-	// TODO: don't assume the name is a mirror?
-	r := &Remote{}
-	r.Name = name
-	r.Mirrors = []string{name}
+func NewRemote(name, aciurl, sigurl string) *Remote {
+	r := &Remote{
+		Name:   name,
+		ACIURL: aciurl,
+		SigURL: sigurl,
+	}
 	return r
 }
 
 type Remote struct {
-	Name    string
-	Mirrors []string
-	ETag    string
-	Blob    string
+	Name   string
```

Comment:
My un understandings (also doing #361) were that remote.Name was previously the download url (don't know why it's called name). Now there's the couple ACIURL and SigURL. I think, that for the caching purposes of cas.remote only ACIURL should be enought.
Or my understandings on the purpose of remote are wrong?

Additionally multiple names (as the discovery string composed by app name and labels) can bring to the, sane discovered image.